terraform/api.tf

```diff
@@ -60,3 +60,7 @@ resource "heroku_formation" "api_checksum_worker" {
 data "aws_iam_user" "api" {
   user_name = module.api.heroku_iam_user_id
 }
+
+resource "aws_iam_user" "backup" {
+  name = "backup"
+}
```

Comment:
I wonder if this should go somewhere else, since it's not a permanent addition. Or maybe just a comment would suffice

Reply:
I'll add a comment explaining what this is. I think if we start managing many more IAM user resources, we could split this out into a separate file, but for just the one right now, I think it's ok to stay here.

See 26613e6bf401bcf82b16f9992ae57c9da9eb53b8.